sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -203,6 +203,70 @@ case class GpuTimeSub(start: Expression,
   }
 }
 
+case class GpuDateAddInterval(start: Expression,
+    interval: Expression,
+    timeZoneId: Option[String] = None)
+    extends GpuTimeMath(start, interval, timeZoneId) {
+
+  override def withTimeZone(timeZoneId: String): TimeZoneAwareExpression = {
```

Comment:
not needed now that we don't have timezone

Reply:
This is in DateAddInterval, so need to add this.